### PR TITLE
Fix the category dropdown on the module catalog page

### LINKED ISSUE #27747: Category filter for recommended modules list
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

Go to Admin > Modules > Modules catalog, you'll have a list of recommended modules in different categories. In the top right corner, try to filter with a category in the dropdown, the items in the list don't change

Capture d’écran de 2022-02-21 09-07-50

### Expected behavior

The items list must change depending on the category selected

### Steps to reproduce

1 - Go to Admin > Modules > Modules catalog

2 - Click on a **Product Page** or any other category in the categories filter

3 - See error > the filter doesn't work! The modules not matching this category must be removed from the list

### PrestaShop version(s) where the bug happened

1.7.8.3

### PHP version(s) where the bug happened

_No response_

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by sowbiba**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | See #27747
| Type?             | bug fix
| Category?         | FO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #27747
| How to test?      | Go to Admin > Modules > Modules catalog, you'll have a list of recommended modules in different categories. In the top right corner, try to filter with a category in the dropdown, the list will be reduced to the modules matching this category
| Possible impacts? | -

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/27748)

### CODE REVIEW

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories_catalog.html.twig [hunk 1]
```diff
@@ -25,22 +25,21 @@
 <div class="ps-dropdown dropdown btn-group bordered mb-1">
   {% for menu in topMenuData %}
     {% set refMenu = menu.refMenu %}
-    <div id="{{refMenu}}" class="dropdown-label" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+    <div id="{{refMenu}}" class="dropdown-label " data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```
Comment (matthieu-rolland) [inline src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories_catalog.html.twig]:
```suggestion
    <div id="{{refMenu}}" class="dropdown-label" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
```

or was there a reason for adding a space ?

**Review by matthieu-rolland — COMMENTED** [review]
One small feedback, lgtm otherwise 👍

File: src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories_catalog.html.twig (hunk 1 above)
Comment (sowbiba) [reply src/PrestaShopBundle/Resources/views/Admin/Module/Includes/dropdown_categories_catalog.html.twig]: No reason at all :D :D Just forgetted
Nice catch

**Review by jolelievre — APPROVED** [review]
Thanks @sowbiba

**Review by matthieu-rolland — APPROVED** [review]
lgtm

**Review by marwachelly — APPROVED** [review]
Hello,

PR is tested and the filter works correctly now. PR is QA approuved.
https://watch.screencastify.com/v/HhX9xuhfZAWStw1d9seq

Thanks,

**Comment by Progi1984** [comment]
Thanks @sowbiba & @marwachelly

**Pull request merged**